modules/msisensor2/msi/main.nf

```diff
@@ -0,0 +1,48 @@
+process MSISENSOR2_MSI {
+    tag "$meta.id"
+    label 'process_med'
+
+    conda (params.enable_conda ? "bioconda::msisensor2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE':
+        'biocontainers/msisensor2:0.1_cv1' }"
+
+    input:
+    tuple val(meta), path(tumour_bam), path(tumour_bam_index), path(normal_bam), path(normal_bam_index), path(intervals)
```

Comment:
Can we have `tumor` instead of `tumour`, to be more coherent with the rest of the tools

Reply:
We have the same issue with some tools that use `gender` and some `sex`...

Reply:
it's a mess.

Reply:
it would also be awesome if we could all agree on an order: tumor first then the normal or the other way around to reduce the amount of channel ordering operations we need to do . But I am afraid this is wishful thinking and not a hill i am willing to die on 😆

Reply:
At some point, I'll have a look at all modules and be sure we only use the right keywords (at least for tumor)